test/functional/roles_test.js

```diff
@@ -0,0 +1,29 @@
+Feature('Admin Web - Roles');
+
+Scenario('add all the roles', I => {
+  I.amOnPage(`https://admin-web-div-ccd-definitions-pr-${process.env.CHANGE_ID}.service.core-compute-preview.internal`);
+  I.see('Sign in');
+  I.fillField('username', 'ccd-importer@server.net');
+  I.fillField('password', 'Password12');
```

Comment:
I realise these are plain text, but they're already in the public domain https://github.com/search?q=org%3Ahmcts+ccd-importer%40server.net&type=Code